jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -48,13 +48,21 @@ done
 # Fetch k8s logs
 fetch_k8s_logs "management_cluster" "/home/airshipci/.kube/config"
 
-mkdir -p "${LOGS_DIR}/${CONTAINER_RUNTIME}"
+
+# Fetch Ironic containers logs before pivoting to the target cluster
+CONTAINER_LOGS_DIR="${LOGS_DIR}/${CONTAINER_RUNTIME}/before_pivoting"
+mkdir -p "${CONTAINER_LOGS_DIR}"
+cp -r /tmp/"${CONTAINER_RUNTIME}"/* "${CONTAINER_LOGS_DIR}"
```

Comment:
What is inside the /tmp/$CONTAINER_RUNTIME/ dir? Why didn't we have a similar copy statement before

Reply:
> What is inside the /tmp/$CONTAINER_RUNTIME/ dir? 

It is Ironic container logs that fetch in [here](https://github.com/Nordix/metal3-dev-env/blob/0ac2fd9d87c94edb60ac82c4c7631e5f729d7886/vm-setup/roles/v1aX_integration_test/tasks/move.yml#L23) before we pivot.

> Why didn't we have a similar copy statement before

Because we didn't know that initial logs would be lost after pivoting until @dtantsur found it out.